docs/source/reference/rest.md

```diff
@@ -1,47 +1,52 @@
-(rest-api)=
```

Comment:
This link target should stay

Reply:
It seems a commit for another doc authenticators-users-basics.md was added here, I've resolved them and it has been merged.